src/pages/Admin/Charity/EditProfile/Form.tsx

```diff
@@ -2,6 +2,7 @@ import { PropsWithChildren } from "react";
 import { Link } from "react-router-dom";
 import { FormValues as FV } from "./types";
 import { UNSDG_NUMS } from "types/lists";
+import Toggle from "pages/Launchpad/common/Toggle";
```

Comment:
```suggestion
import Toggle from "components/Toggle";
```

kindly move this since its now used in `Admin` and `Launchpad`

Reply:
done